pages/common/indent.md

```diff
@@ -0,0 +1,16 @@
+# indent
+
+> Change the appearance of a C/C++ program by inserting or deleting whitespace.
+> More information: <https://www.gnu.org/software/indent/>.
+
+- Format C/C++ source according to the Linux style guide:
+
+`indent -linux {{source1.c}} > {{source1.out}}`
```

Comment:
Please use the long argument format. And we use a path to file placeholder.
```suggestion
`indent --linux-style {{path/to/source.c}} > {{path/to/indented_source.c}}`
```
I read the docs, but I feel that we should keep the `.c` extension for the output file.

Reply:
Updated to use long argument format and path to file placeholders.